### PR TITLE
added myid support for cluster setup

### PR DESCRIPTION
**Pull request by filiptepper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,14 @@ An Ansible role for installing [Apache Zookeeper](http://zookeeper.apache.org).
 - `zookeeper_version` - Zookeeper version.
 - `zookeeper_cloudera_distribution` - Cloudera distribution version (default: `cdh5.4`)
 - `zookeeper_conf_dir` - Configuration directory for Zookeeper (default: `/etc/zookeeper/conf`)
-- `zookeeper_data_dir`: Data directory for Zookeeper (default: `/var/lib/zookeeper`)
```
Comment (hectcastro) [inline README.md]: Thanks for fixing these.

File: tasks/main.yml [hunk 2]
```diff
@@ -28,5 +28,13 @@
   notify:
     - Restart Zookeeper
 
+- name: Configure JAVA_OPTS
```
Comment (hectcastro) [inline tasks/main.yml]: I was unable to get this addition to work until I set `zookeeper_java_opts` to:

```yaml
zookeeper_java_opts: "export SERVER_JVMFLAGS=-Xmx2048m"
```

Through the process of figuring that out, I noticed that the `/etc/default/zookeeper` file contains a commented out reference to `SERVER_JVMFLAGS`. Does it make more sense to just template out vs. creating `java.env`?

Comment (filiptepper) [inline tasks/main.yml]: Shoot, my bad, sorry for that. I think I may have left out some of the stuff in this commit. I'll get back to you once I review it, sorry again for the trouble!

**Comment by filiptepper** [comment]
I have submitted an improved PR #5.

**Pull request closed without merging**